ouroboros-network/demo/chain-sync.hs

```diff
@@ -212,6 +218,11 @@ serverPingPong = do
         codecPingPong
         (pingPongServerPeer pingPongServerStandard)
 
+    serverK rspFn = do
+        let (MiniProtocolResponderControl ppResult) = rspFn PingPong0
```

Comment:
it seems to be a common pattern to unpack the `MiniProtocolResponderControl` newtype (or `MiniProtocolInitiatorControl`).  Maybe provide a destructor:
```
newtype MiniProtocolInitiatorControl = MiniProtocolInitiatorControl { getMiniProtocolInitiatorControl :: ... }
```
```

Reply:
Will address.